org.opent2t.sample.hub.superpopular/com.hue.hub/test/mockHueHub.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Curious @ Arjun-  i'd noticed none of the files here had a copyright etc header -- is it because we are opensource? Even then, do we need some headers for our code in general at all? licensing info etc ?